GVRf/Framework/framework/src/main/jni/objects/textures/base_texture.h

```diff
@@ -98,41 +71,18 @@ class BaseTexture: public Texture {
     virtual void runPendingGL() {
```

Comment:
Is it possible to keep getId() and runPendingGL() unchanged while adding the new thread?